spec/helpers/application_helper_spec.rb

```diff
@@ -66,12 +66,12 @@
       end
 
       it 'should use the environment variable' do
-        ENV['OSEM_NAME'] = Faker::Company.name
-        expect(nav_root_link_for(nil)).to match ENV['OSEM_NAME']
+        ENV['OSEM_NAME'] = Faker::Company.name + "'"
```

Comment:
So we will only add a quote at the end?

Reply:
I wanted to make sure there was _always_ an apostrophe in the test string, as that was the character Faker commonly spits out, which was being html encoded, but it sometimes doesn't. So, forcing an apostrophe at the end of the string makes the test behave consistently - it _always_ has to handle html encoded characters :)